src/osvr/Common/CommonComponent.cpp

```diff
@@ -35,8 +35,24 @@ namespace osvr {
 namespace common {
     namespace messages {
 
+        // These messages all must match the ones in VRPN exactly.
+        /// @todo add tests to verify that these message identifiers match VRPN
+        /// string constants.
+
```

Comment:
OK, adding tests for the new stuff. The existing stuff is harder to test, but test coverage isn't decreasing with this pull request once updated.